NOTES.md

```diff
@@ -0,0 +1,5 @@
+- aslink87: set color theme css styles in globals.css if you'd like to change global styles for specific elements such as headings
```

Comment:
Some of this can go in the README file and some can be moved to projects if you want to track / have a discussion.